client/my-sites/importer/importing-pane.jsx

```diff
@@ -194,9 +195,22 @@ export default React.createClass( {
 	},
 
 	render: function() {
-		const { site: { ID: siteId, name: siteName, single_user_site: hasSingleAuthor } } = this.props;
-		const { importerId, errorData = {}, customData } = this.props.importerStatus;
+		const {
+			importerStatus: {
+				importerId,
+				errorData = {},
+				customData
+			},
+			mapAuthor: mapAuthorFor,
```

Comment:
@mcsf I changed to prevent a naming clash with the imported `mapAuthor` action creator. I realize that I didn't need to do this, but I wanted to try and prevent confusion for those reading the code.

do you think I should _not_ rename it?